.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# See https://help.github.com/articles/about-codeowners/
+# for more info about CODEOWNERS file
+
+# It uses the same pattern rule for gitignore file
+# https://git-scm.com/docs/gitignore#_pattern_format
+
+* @anselmbradford @cfarm @contollini @jimmynotjim @Scotchester
```

Comment:
You've got an extra `l` in @contolini.

Reply:
If I had a nickel...

Reply:
@contolini ha, maybe you should change your name to match our mental model of how it's spelled ;) I've fixed it and amended the original commit.